src/Enforce.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import useEnforcer from './useEnforcer';
+
+interface EnforceProps {
+  deny?: boolean;
+  sub?: string;
+  obj: string;
+  act: string;
+}
+const Enforce: React.FC<EnforceProps> = (props) => {
+  const { deny = false, obj, act, children } = props;
+  const enforcer = useEnforcer();
+
+  const sub = props.sub || enforcer.sub!;
+  const enforceRes = enforcer.enforce(sub, obj, act);
+  const shouldRender = (enforceRes && !deny) || (!enforceRes && deny);
+
+  return <>{shouldRender && children}</>;
```

Comment:
Maybe we should have some discussion on API.
I'm the author of `vue-authz`. I make the API as same as `casbin.js`.
We should try to make the API unified in different version.

Reply:
Let's unify the API as next step.